src/order/filter/n_ary.lean

```diff
@@ -44,6 +44,29 @@ def map₂ (m : α → β → γ) (f : filter α) (g : filter β) : filter γ :=
       (image2_subset (inter_subset_right _ _) $ inter_subset_right _ _).trans ht⟩,
   end }
 
+lemma map₂_eq_map_prod (m : α → β → γ) (f : filter α) (g : filter β) :
```

Comment:
Can you add the `set.image2` version as well?

Reply:
Sure. Should I make a separate PR? Otherwise, what do I change feat(order/filter/n_ary) to?

Reply:
You can keep the same title. This is really the continuation.

Reply:
Hmm... isn't set.image_prod already the set version though?

Reply:
Then your lemma should look the same.

Reply:
OK, I'll flip the equality.

Reply:
OK, I did more than just flip the equality. Is it better?

Reply:
I will consider you silence as approval.